[limesurvey] Error during build

Error trying to build during SSL hardening. Not sure if it's related the SSL stuff, looks like maybe upstream issue? Reporting anyway for followup.

Testing Tracking here (https://github.com/turnkeylinux/common/pull/42#issuecomment-132284465)


#### Error Below
```
# apply the common removelists
fab-apply-removelist /turnkey/fab/common/removelists/mysql build/root.patched;  
warning: entry does not exist: var/lib/mysql/ib_logfile*
fab-apply-removelist /turnkey/fab/common/removelists/turnkey build/root.patched; 
warning: entry does not exist: etc/apt/trusted.gpg~
warning: entry does not exist: etc/apt/trusted.gpg.d/debian.gpg~
warning: entry does not exist: etc/apt/trusted.gpg.d/turnkey.gpg~
# apply the product-local root overlay
if [ -d overlay ]; then fab-apply-overlay overlay build/root.patched; fi
# run the product-local configuration scripts
$(call run-conf-scripts, conf.d)
fab-chroot build/root.patched --script conf.d/downloads
+ VERSION=206plus-build150729
+ URL=http://download.limesurvey.org/latest-stable-release/limesurvey206plus-build150729.tar.gz
+ dl http://download.limesurvey.org/latest-stable-release/limesurvey206plus-build150729.tar.gz /usr/local/src
+ '[' http://127.0.0.1:8124 ']'
+ PROXY='--proxy http://127.0.0.1:8124'
+ cd /usr/local/src
+ curl -L -f -o limesurvey-205.tar.gz --proxy http://127.0.0.1:8124 http://download.limesurvey.org/latest-stable-release/limesurvey206plus-build150729.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (22) The requested URL returned error: 500 Internal Server Error
/usr/share/fab/product.mk:476: recipe for target 'build/stamps/root.patched' failed
make: *** [build/stamps/root.patched] Error 22
```

--- a/changelog
+++ b/changelog
@@ -6,12 +6,12 @@ turnkey-limesurvey-14.0 (1) turnkey; urgency=low
 
   * Upstream source component versions:
 
-    limesurvey      206plus-build150729
+    limesurvey      206plus-build150825
 
   * Note: Please refer to turnkey-core's changelog for changes common to all
     appliances. Here we only describe changes specific to this appliance.
 
- -- Jeremy Davis <jeremy@turnkeylinux.org>  Thu, 30 Jul 2015 01:08:23 +1000
+ -- Jeremy Davis <jeremy@turnkeylinux.org>  Thu, 26 Aug 2015 10:25:08 +1000
 
 turnkey-limesurvey-13.0 (1) turnkey; urgency=low
 

--- a/conf.d/downloads
+++ b/conf.d/downloads
@@ -5,7 +5,7 @@ dl() {
     cd $2; curl -L -f -o limesurvey-205.tar.gz $PROXY $1; cd -
 }
 
-VERSION="206plus-build150729"
+VERSION="206plus-build150812"
 URL="http://download.limesurvey.org/latest-stable-release/limesurvey${VERSION}.tar.gz"
 
 dl $URL /usr/local/src

--- a/conf.d/downloads
+++ b/conf.d/downloads
@@ -5,7 +5,7 @@ dl() {
     cd $2; curl -L -f -o limesurvey-205.tar.gz $PROXY $1; cd -
 }
 
-VERSION="206plus-build150812"
+VERSION="206plus-build150825"
 URL="http://download.limesurvey.org/latest-stable-release/limesurvey${VERSION}.tar.gz"
 
 dl $URL /usr/local/src

--- a/overlay/etc/apache2/sites-available/limesurvey.conf
+++ b/overlay/etc/apache2/sites-available/limesurvey.conf
@@ -8,7 +8,6 @@ ServerName localhost
 
 <VirtualHost *:443>
     SSLEngine on
-    SSLCertificateFile /etc/ssl/certs/cert.pem
     ServerAdmin webmaster@localhost
     DocumentRoot /var/www/limesurvey/
 </VirtualHost>

